src/components/expandable-row.vue

```diff
@@ -20,12 +20,12 @@ except according to the terms contained in the LICENSE file.
     </div>
 
     <div class="button-cell">
-      <a href="javascript:void(0)" role="button" @click.prevent="toggleExpanded">
+      <button type="button" class="btn btn-link" @click="toggleExpanded">
```

Comment:
I had forgotten that we have `.btn-link` and that it's useful for cases like this. We have a similar `<a role="button">` in `ProjectHomeBlock`, which we should probably replace with a `<button class="btn btn-link">` at some point.